evap/static/ts/tsconfig.compile.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "rootDir": "./src",
+        "outDir": "../../static/js",
+        "incremental": true,
+        "tsBuildInfoFile": ".tsbuildinfo.json"
```

Comment:
I had seldom cases where this build file got out-of-sync of my local file system. E.g. when running `ts compile`, then deleting the generated `.js` (but keep `.tsbuildinfo.json`), running `ts compile` again won’t generate the files. Deleting the `.tsbuildinfo` should fix these cases. I think it should normally not happen – unless you crazely rebase a certain branch named `typescript` unusually often – but it could be one of these places which is a pain troubleshoot. Of course, incremental compilation gives a time boost (in my case, -50% if nothing has changed). How do you feel about it?

Reply:
I would say its fine, add an entry to the readme that this can happen and how to fix it (probably you can override incremental build with a command line flag?)

I think ideally `ts compile` would print when it compiles a file, so you would wonder why your in-browser-debugger says the code doesn't change, then you'd see that no compilation happens, and then you'd quickly try throwing away cached stuff and building it from scratch (I mean, e.g. npm, I'm used to `rm -rf node_modules && npm ci` because in my experience, that stuff just does not work the way it should)

Reply:
Hm, good point with the `.gitignore`, dunno how I encountered this situation by accident. I couldn’t find a way to print out compiled files with `tsc`, so I guess having a command argument which either deletes `.tsbuildinfo` or overwrites the configuration is a sensible way here.

Reply:
Deleting the `.tsbuildinfo` was easier and has the benefit that the next call uses incremental compilation. `--fresh` is available for both `ts compile` and `ts test`